rtengine/iptransform.cc

```diff
@@ -519,6 +519,21 @@ bool ImProcFunctions::transCoord (int W, int H, const std::vector<Coord2D> &src,
             pLCPMap->correctDistortion(x_d, y_d, w2, h2);
         }
 
+        x_d /= params->perspective.camera_scale;
+        y_d /= params->perspective.camera_scale;
+        if (params->perspective.camera_defish) {
+            double const focal_source = params->perspective.camera_focal_length * maxRadius * 0.0462;
```

Comment:
We use `const` on the left throughout the code. Please change `double const` to `const double`. Thanks.